automatic/selenium.powershell/tools/chocolateyBeforeModify.ps1

```diff
@@ -0,0 +1,8 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$ModuleName = "Selenium"
+
+# The module shouldn't be present in the Chocolatey session, but...
+if (Get-Module $ModuleName) {
+  Remove-Module $ModuleName -Force -ErrorAction SilentlyContinue
+}
```

Comment:
The module shouldn't be removed as part of a before modify script.
Instead, extract it out to a helper file that is called from both the `chocolateyInstall` and `chocolateyUninstall` script.

Reply:
I've just taken this out completely as there shouldn't be a scenario where it's ever loaded in the Chocolatey PowerShell session, nor will removing it actually acomplish anything for this scenario as it would still have the DLLs locked.